api/src/wfl/module/batch.clj

```diff
@@ -30,6 +94,18 @@
     (jdbc/update!        tx :workload {:items table} ["id = ?" id])
     [id table]))
 
+(defn pre-v0.4.0-load-workload-impl
```

Comment:
What is this function's eventual fate, guessing it's here for backwards compatibility?  Is there a known condition that would allow us to delete it?

Reply:
yes I think there is one.
In the future I propose that we add a "InputsListSource" - a source that's literally a list of workflow inputs:
```json
...
"source": {
    "name": "Inputs List",
    "inputs": [{
        "sample_name": "Sample",
        ...
    },
    ...
    ]}},
...
```

This would allow us to "batch" workflows in the way we currently do but recast into this source->executor->sink model.